rq/job.py

```diff
@@ -1507,3 +1566,16 @@ def __init__(self, max: int, interval: Union[int, List[int]] = 0):
 
         self.max = max
         self.intervals = intervals
+
+
+class Callback:
+    def __init__(self, func: Callable[..., Any], timeout: Optional[Any] = None):
```

Comment:
`timeout` should be an int or float.

Reply:
ditching support of passing a string? `parse_timeout` at the moment handles int,float and strings like `1m`

Reply:
Sorry, totally forgot that we handle strings as timeout too